python/tvm/relay/backend/contrib/ethosu/tir/scheduler.py

```diff
@@ -15,17 +15,17 @@
 # specific language governing permissions and limitations
 # under the License.
 # pylint: disable=invalid-name, unused-argument
-"""Different schedulers for Arm(R) Ethos(TM)-U NPU"""
+"""Scheduling for Arm(R) Ethos(TM)-U NPU."""
 import tvm
 
 
-def schedule(te_graph, const_dict, cascader=None):
-    """Schedule a TE graph for NPU compilation.
+def schedule(cached_func, const_dict, cascader=None):
+    """Schedule a CachedFunc for NPU compilation.
 
     Parameters
     ----------
-    te_graph
-        The TE graph to schedule.
+    cached_func : CachedFunc
```

Comment:
Heads up we're trying to move both the TECompiler and CachedFunc structures to be internal only. I think the idea is all the cross-reference stuff accumulated in CachedFunc would be captured as 'official' attributes on the call or defn. So obviously we'll need to include this in that refactor.